packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -114,9 +121,21 @@ class CollapsibleMenuView extends React.Component {
   }
 
   handleResize(width) {
-    const menuButtonWidth = this.menuButton.getBoundingClientRect().width;
-    const availableWidth = width - menuButtonWidth;
     const childrenArray = React.Children.toArray(this.props.children);
+    const menuButtonWidth = childrenArray.length > 1 ? this.menuButton.getBoundingClientRect().width : 0;
+    const menuButtonContainerWidth = this.menuButton.parentElement.getBoundingClientRect().width;
+    let availableWidth = width - menuButtonWidth;
+    this.isContainerWidthUsedtoResize = false;
+    // if no wrapper is used on top of collapsiblemenuview, use observer width value to calculate availableWidth space,
+    // or use menuButtonContainerWidth value to calculate availableWidth space
+    if (width < menuButtonContainerWidth) {
+      availableWidth = menuButtonContainerWidth - menuButtonWidth;
+      this.isContainerWidthUsedtoResize = true;
+    }
+    // to calculate available space when resized only when menuButtonContainerWidth is used
+    if ((window.innerWidth - menuButtonWidth) < availableWidth) {
+      availableWidth = window.innerWidth - menuButtonWidth;
```

Comment:
will this be having -ve value ..?? I think we can use Math.Abs() if we are not sure edge case where it could be -ve

Reply:
Added Math.abs to avoid -ve values.
c66f453f784054782522849ab28b84da96be7655
Thanks